src/V3Options.cpp

```diff
@@ -1330,47 +1330,6 @@ void V3Options::parseOptsList(FileLine* fl, const string& optdir, int argc,
     DECL_OPTION("-O2", CbCall, [this]() { optimize(2); });
     DECL_OPTION("-O3", CbCall, [this]() { optimize(3); });
 
-    DECL_OPTION("-O", CbPartialMatch, [this, fl](const char* optp) {
-        // Optimization, e.g. -O1rX
-        // LCOV_EXCL_START
-        fl->v3warn(DEPRECATED, "Option -O<letter> is deprecated. "
-                               "Use -f<optimization> or -fno-<optimization> instead.");
-        for (const char* cp = optp; *cp; ++cp) {
-            const bool flag = std::isupper(*cp);
-            switch (std::tolower(*cp)) {
-            case '0': optimize(0); break;
-            case '1': optimize(1); break;
-            case '2': optimize(2); break;
-            case '3': optimize(3); break;
```

Comment:
I was going to comment these need to stay but this seems to have been dead code per the above!